cmd/mesh/shared.go

```diff
@@ -40,31 +58,42 @@ func getWriter(outFilename string) (*os.File, error) {
 	return writer, nil
 }
 
-func checkLogsOrExit(args *rootArgs) {
-	if err := configLogs(args); err != nil {
-		_, _ = fmt.Fprintf(os.Stderr, "Could not configure logs: %s", err)
-		os.Exit(1)
-	}
+type logger struct {
+	logToStdErr bool
+	stdOut      io.Writer
+	stdErr      io.Writer
 }
-func configLogs(args *rootArgs) error {
-	opt := log.DefaultOptions()
-	if !args.logToStdErr {
-		opt.ErrorOutputPaths = []string{logFilePath}
-		opt.OutputPaths = []string{logFilePath}
+
+func newLogger(logToStdErr bool, stdOut, stdErr io.Writer) *logger {
+	return &logger{
+		logToStdErr: logToStdErr,
+		stdOut:      stdOut,
+		stdErr:      stdErr,
 	}
-	return log.Configure(opt)
 }
 
 // TODO: this really doesn't belong here. Figure out if it's generally needed and possibly move to istio.io/pkg/log.
```

Comment:
is the TODO part still a concern?

Reply:
Yes :-/ this is one of the TODOs I want to address when we get the ball rolling on 1.4.
There are about 30 others!